hydra-web/src/js/modules/alias.js

```diff
@@ -131,7 +131,17 @@ function(
                     "bVisible":true,
                     "bSearchable":true,
                     "mRender":function(val,type,data){
-                        return val.join(",");
+                        var length = val.length;
+                        if(length == 1) {
+                            return "<a href='#jobs/"+encodeURIComponent(val)+"/conf'>"+val+"</a>";
+                        } else if(length > 1) {
+                            var list = "";
+                            for(i in val)
+                                list += "<a href='#jobs/"+encodeURIComponent(val[i])+"/conf'>"+val[i]+"</a>&nbsp;&nbsp;";
+                            return list;
+                        } else {
+                            return "";
+                        }
```

Comment:
There is no need to handle an array of 1 element separately than more than one element, this only complicates the code.